src/resource/solidDataset.ts

```diff
@@ -871,14 +872,31 @@ export function getContainedResourceUrlAll(
 }
 
 /**
- * Given a {@link SolidDataset} representing a Container (see {@link isContainer}), verify that
- * all its containùent claims are valid. Containment of a resource is invalid if it doesn't
- * respect slash semantics, see https://solidproject.org/TR/protocol#resource-containment for
- * more details.
+ * Given a {@link SolidDataset} representing a [Container](https://solidproject.org/TR/protocol#resource-containment)
+ * (see {@link isContainer}), verify that all its containment claims are valid.
+ *
+ * Containment of a resource is valid if it respects
+ * [slash semantics](https://solidproject.org/TR/protocol#uri-slash-semantics). For the
+ * container at https://example.org/container/, the following resources are valid:
+ *  - https://example.org/container/resource
+ *  - https://example.org/container/subcontainer/
+ *
+ * The following resources are invalid:
+ *  - https://example.org/container/resource/invalid (not a direct child resource)
+ *  - https://example.org/container2 (not a child resource)
+ *  - https://domain2.example.org/container/resource (not a direct child resource)
  *
  * Resources for which containment is invalid are not included in the result set returned by
  * {@link getContainedResourceUrlAll}.
  *
+ * The ESS will always include containment triples that respect
```

Comment:
Do we want to finesse this to make it less like we're just  ESS-centric vs we're supporting Solid-compliant servers?  Like

> ESS, conforming to the Solid Protocol,  respects [slash semantics](https://solidproject.org/TR/protocol#uri-slash-semantics) for its containment triples.  As such, the function returns true for containers fetched from ESS.